packages/liveblocks-client/src/room.ts

```diff
@@ -258,7 +259,8 @@ export type State<TPresence extends JsonObject> = {
       storageUpdates: Map<string, StorageUpdate>;
     };
   };
-  offlineOperations: Map<string, Op>;
+  offlineOperations: Map<string | undefined, Op>;
+  //                              ^^^^^^^^^ NOTE: Bug? Unintended?
```

Comment:
Here's another case where the unit test suite pointed out a case where we annotate that this Map only ever contains string keys, but in reality it also relies on `.set(undefined, ...)` and `.get(undefined)` calls. This may need some investigation.

Reply:
Operations created by LiveList.clear were missing an `opId`.

Reply:
Ah! Glad we found this bug. Should `opId` be a mandatory property on those operations? Or are there valid reasons for an `opId` to be missing?

Reply:
Operations sent over the network always contain an `opId` but they do not while inside the undo/redo stack. We assign an `opId` to operations from the undo/redo stack just before sending them to the server. There is probably a better way to model it (tuple `[opId, Op]`?) while reworking the op structure.